### PR TITLE
Add initial Hadoop Plugin example project

### PR DESCRIPTION
**Pull request by convexquad**

### CODE REVIEW

**Comment by convexquad** [comment]
Closes #117

**Comment by convexquad** [comment]
@nntnag17 and @pranayhasan Can you guys look through my example project and see if there's anything you think could be improved?

**Comment by convexquad** [comment]
@nntnag17 and @pranayhasan an easier way to review this PR might be to look through the example project in my personal GitHub at https://github.com/convexquad/linkedin-gradle-plugin-for-apache-hadoop/tree/master/example-project

**Comment by pranayhasan** [comment]
@convexquad Thanks, I will review today!

File: example-project/build.gradle [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      into "text"
+    }
+  }
+
+  zip("azkabanProd") {
+    from("resources-prod") {
+      into "text"
+    }
+  }
+}
```
Comment (pranayhasan) [inline example-project/build.gradle]: Lets have a dir structure for resources by having resources/dev, resources/prod etc

File: example-project/src/main/profiles/abain.gradle [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Each developer on your team can override the default definition set declared at the top of the
+// workflows.gradle file with their own values for testing.
+definitionSet defs: [
+    hdfsHomeDir: "/user/abain",
+    notifyEmail: "myEmail@myEmailDomain.com",
+    userToProxy: "abain"
```
Comment (pranayhasan) [inline example-project/src/main/profiles/abain.gradle]: Can we make this generic, maybe **myname**?

**Comment by pranayhasan** [comment]
This example project is very well commented out and makes it a lot easier to understand using Hadoop Plugin. A big thanks @convexquad

**Comment by convexquad** [comment]
@pranayhasan I fixed the issues you mentioned in your review - thanks!

**Pull request merged**